### PR TITLE
Chat window enhance

### PR DESCRIPTION
**Pull request by dahvreinhart**
- properly spaces and formatted chat window buttons
- glyphicon in button to make function clearer
- chat page is now responsive with formatting being the same through window size transitions
- chat window has been made bigger, and the text size increase for better readability

### CODE REVIEW

File: client/styles/simpleChatStyleOverwrite.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.box-body {
```
Comment (madeline-petersen) [inline client/styles/simpleChatStyleOverwrite.css:1]: If we're only using these styles in the chat, could move them to SpaceFrame/index.css?

Comment (dahvreinhart) [reply client/styles/simpleChatStyleOverwrite.css:1]: they must be in the client directory apparently in order to properly override

Comment (dahvreinhart) [reply client/styles/simpleChatStyleOverwrite.css:1]: we tried it the other way with no luck. This way we found on a tutorial.

Comment (madeline-petersen) [reply client/styles/simpleChatStyleOverwrite.css:1]: 👍

**Pull request merged**